modules/nf-core/sentieon/tnhaplotyper2/main.nf

```diff
@@ -82,7 +82,7 @@ process SENTIEON_TNHAPLOTYPER2 {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.vcf.gz
+    echo | gzip > ${prefix}.vcf.gz
     touch ${prefix}.vcf.gz.tbi
     touch ${prefix}.contamination_data.tsv
     touch ${prefix}.orientation_data.tsv
```

Comment:
Shouldn't the optional output only present if the command is set to true ?

Reply:
I guess